lightning/src/routing/network_graph.rs

```diff
@@ -98,6 +121,28 @@ impl<C: Deref, L: Deref> NetGraphMsgHandler<C, L> where C::Target: ChainWatchInt
 	pub fn read_locked_graph<'a>(&'a self) -> LockedNetworkGraph<'a> {
 		LockedNetworkGraph(self.network_graph.read().unwrap())
 	}
+
+	/// Enqeues a message send event for a batch of short_channel_ids
+	/// in a task.
+	fn send_channels_query(&self, task: &mut GossipQueryShortChannelIdsTask) {
```

Comment:
nit: Maybe a better name would be `finalize_query_short_ids`, as otherwise you may think it can used for `SendChannelRangeQuery`